includes/Data/Fonts.php

```diff
@@ -0,0 +1,229 @@
+<?php
```

Comment:
What do you think of a structure where `Colors.php` and `Fonts.php` go into `Themes.php` or a folder called `Themes` similar to controllers that contains all 3 files. As we give more customisation for a theme we will have more of these in the parent folder.

Reply:
Makes sense, I'll look into that.